block_height: 0.039305143873686596
block_raw_z: -0.08189160903254694
block_z: -0.06189160903254694

position_x_dif: 0.18886987978937214
position_y_dif: 0.1027965051939757

table_raw_z: -0.12119675290623354
table_z: -0.10119675290623353